packages/bpk-component-floating-notification/src/__snapshots__/BpkFloatingNotification-test.tsx.snap

```diff
@@ -3,7 +3,7 @@
 exports[`BpkFloatingNotification should render correctly 1`] = `
 <DocumentFragment>
   <div
-    class="bpk-floating-notification bpk-floating-notification--appear bpk-floating-notification--appear-active"
+    class="bpk-floating-notification"
```

Comment:
Do you know why the classnames have changed here? Looks like it might not be appearing anymore?